File: Pascal/Dcoder/TracysLove.pas
//fpc 3.0.0

program of_example;
var
		a, b, c, d: integer;
begin
		while not eof do begin
			read(a);
			read(b);
			c := Abs(a-b);
			d := Abs(a+b);
			if (c = 6) or (d = 6) then
				writeln('Love')
			else
				writeln('Hate');
		end
end.

File: Java/Dcoder/TracysLove.java
import java.util.*;


/*Please dont change class name, Dcoder 
 and class must not be public*/

//Compiler version JDK 1.8


class Dcoder
{ 
	public static void main(String args[])
	{
		int a,b;
		Scanner scn = new Scanner(System.in);
		a = scn.nextInt();
		b = scn.nextInt();
		if (Math.abs(a-b) == 6 || Math.abs(a+b) == 6) {
			System.out.println("Love");
		}
		else {
			System.out.println("Hate");
		}
	}
}
